sources/packages/web/src/views/supporting-user/Dashboard.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <full-page-container>
-    <formio
+    <formio-container
```

Comment:
In this case, I do not see a benefit in doing change. What was the motivation for the change?

Reply:
I'm not sure about the difference between <formio> and <formio-container>. I thought we were replacing them as @ann-aot  asked me to do that.

Reply:
@andrewsignori-aot ok. So, is it resolved?